lib/routes/user/home/payments_filter.dart

```diff
@@ -151,4 +149,37 @@ class PaymentsFilterState extends State<PaymentsFilter> {
     }
     return [PaymentType.RECEIVED, PaymentType.DEPOSIT, PaymentType.SENT, PaymentType.WITHDRAWAL];
   }
+
+  IconButton _buildExportButton(BuildContext context) {
+    return widget._paymentsModel.paymentsList.isNotEmpty
+        ? IconButton(
+            icon: ImageIcon(
+              AssetImage("src/icon/export.png"),
+              color: Colors.white,
+              size: 24.0,
+            ),
+            onPressed: () => _exportPayments(context),
+          )
+        : IconButton(
+            icon: ImageIcon(
+              AssetImage("src/icon/export.png"),
+              color: Theme.of(context).disabledColor,
+              size: 24.0,
+            ),
+          );
+  }
+
+  Future _exportPayments(BuildContext context) async {
+    var action = ExportPayments();
+    widget._accountBloc.userActionsSink.add(action);
+    Navigator.of(context).push(createLoaderRoute(context));
+    action.future.then((filePath) {
+      Navigator.of(context).pop();
+      final RenderBox box = context.findRenderObject();
+      ShareExtend.share(filePath, "file", sharePositionOrigin: box.localToGlobal(Offset.zero) & box.size);
+    }).catchError((err) {
+      Navigator.of(context).pop();
+      Scaffold.of(context).showSnackBar(new SnackBar(content: new Text("Failed to export payment list.")));
```

Comment:
Better use `FlushBar` as we moved to it across the board.

Reply:
[Replaced](https://github.com/breez/breezmobile/pull/203/commits/4ba1eb2dfac47c2627d75fa6b6ab1dbb1356ef26)